src/ddocs/views/pagination.rst

```diff
@@ -123,8 +123,60 @@ Or in a pseudo-JavaScript snippet:
         page.display_link('next');
     }
 
-Paging
-======
+
+Paging (New Way)
```

Comment:
Could we give these proper names, rather than simply "old" vs "new"?

Perhaps the old way could be called "View-based Paging" for example?